test/e2e/service.go

```diff
@@ -309,17 +309,6 @@ var _ = Describe("Services", func() {
 		}
 		expectNoError(verifyServeHostnameServiceUp(c, host, podNames1, svc1IP, servicePort))
 		expectNoError(verifyServeHostnameServiceUp(c, host, podNames2, svc2IP, servicePort))
-
-		// Remove iptable rules and make sure they come back.
-		By("Remove iptable rules and make sure they come back")
```

Comment:
We don't really want to remove this block - we want to add the KUBE-SERVICES chain to the list and flush that, too.